g2core/marlin_compatibility.cpp

```diff
@@ -406,14 +406,14 @@ stat_t marlin_report_version()
     char buffer[128];
     char *str = buffer;
 
-    str_concat(str, "ok FIRMWARE_NAME:Marlin g2core-");
+    str_concat(str, "ok FIRMWARE_NAME:Marlin g2core-vegetablejedi-");
```

Comment:
Probably leave out the "vegetablejedi" for PR's to the main repo. :grin:

Reply:
Yes. When I did a pull request, it was before I put the phrase, but after I committed it, the phrase went in like that. I suppose I had to work on the branch to pull requests. :-)